node/cn/tracers/internal/tracers/bigram_tracer.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
Please change license descriptions.
You can refer other file license.

Reply:
I change license descriptions of tracer javascript files. Please check those changes @KimKyungup